src/server/routes/__tests__/submissions.unit.spec.js

```diff
@@ -326,4 +326,142 @@ describe('submissions.unit', () => {
       expect(h.code).toHaveBeenCalledWith(500)
     })
   })
+
+  describe('GET /submissions/{submissionId}/activities', () => {
+    const getActivitiesHandler = routes[2].options.handler
+
+    const getFoundSubmissionWithActivities = (activities = []) => ({
+      id: '1',
+      contactId: 'contact-identifier-111',
+      season: '2024',
+      status: 'COMPLETE',
+      source: 'WEB',
+      version: '2024-10-10T13:13:11.000Z',
+      Activities: activities,
+      createdAt: '2024-10-10T13:13:11.000Z',
+      updatedAt: '2024-10-10T13:13:11.000Z'
+    })
+
+    const getActivityMock = () => ({
+      toJSON: jest.fn().mockReturnValue({
+        id: '1',
+        daysFishedWithMandatoryRelease: 1,
+        daysFishedOther: 2,
+        createdAt: '2024-10-10T13:13:11.000Z',
+        updatedAt: '2024-10-10T13:13:11.000Z'
+      })
+    })
+
+    const getActivitiesRequest = () => ({
+      info: {
+        host: 'localhost:3000'
+      },
+      server: {
+        info: {
+          protocol: 'http'
+        }
+      },
+      params: {
+        submissionId: '1'
+      }
+    })
```

Comment:
These are also looking remarkably similar - the only thing that's varying between the `getXXXRequest` is the params or query. Therefore, again, I think they could be defined at the top, with common elements defined in a separate function, something like this:
```
const getRequest = () => ({
  info: {
    host: 'localhost:3000'
  },
  server: {
    info: {
      protocol: 'http'
    }
  }
})
const getPostSubmissionRequest = () => ({
  ...(getRequest()),
  payload: {
    contactId: 'contact-identifier-111',
    season: '2024',
    status: 'INCOMPLETE',
    source: 'WEB'
  }
})
const getContactIdAndSeasonSubmissionRequest = () => ({
  ...(getRequest()),
  query: {
    contact_id: 'contact-identifier-111',
    season: '2024'
  }
})
const getActivitiesRequest = () => ({
  ...(getRequest()),
  params: {
    submissionId: '1'
  }
})
```
Whenever you find yourself copy and pasting code, or can identify areas of commonality in a test file, it's best to see how you can combine them. When test files get as big as this one's getting, it can be unavoidable, but it makes them easier to understand if common functionality is broken out like this.